installer/inventory

```diff
@@ -44,8 +44,10 @@ pg_password=awxpass
 pg_database=awx
 pg_port=5432
 
-# AWX Tunables
-awx_secret_key=aabbcc
+# AWX Secret key
+# It's *very* important that this stay the same between upgrades or you will use the ability to decrypt
+# your credentials
+awx_secret_key=awxsecret
```

Comment:
Just to reiterate about the upgrade path.. the specific string value "awxsecret" is the same as what we were previously writing to the docker image, but now a user will be able to edit this within their customized inventory file. Is that about right?

Reply:
right, I replaced this because it was what we were writing in the docker container itself.